gr-audio/lib/portaudio/portaudio_sink.cc

```diff
@@ -34,6 +34,9 @@
 #include <unistd.h>
 #include <stdexcept>
 #include <string.h>
+#ifdef _MSC_VER
+#include <io.h>
+#endif 
```

Comment:
Did you mean
`#ifdef _WIN32`
if so then I'm good with that, can't test it though (on MinGW) so didn't put it that way to start with